flathunter/sender_apprise.py

```diff
@@ -25,10 +22,14 @@ def process_expose(self, expose):
             url=expose['url'],
             address=expose['address'],
             durations="" if 'durations' not in expose else expose['durations']).strip()
-        self.send_msg(message)
+        self.__send_msg(message)
         return expose
 
-    def send_msg(self, message):
+    def notify(self, message: str):
```

Comment:
Why did you rename ```send_msg``` to ```__send_msg```, only to implement ```notify```, referencing it? 🤔

Reply:
send message is only responsible to send a message, and notify is responsible to notify a message to all subscribers.

we might need to alter a few things in the future before notifying users and if we continue with this, we will end up with a tightly coupled code and need to refactor it in the future. so I did it now before things get complicated.